ui/src/environments/environment.test.ts

```diff
@@ -6,5 +6,5 @@ export const environment: Environment = {
   displayTag: 'Test',
   debug: false,
   enablePublishedWorkspaces: true,
-  enableAccountPages: false
+  enableAccountPages: true
```

Comment:
For simplicity in cherry-picking a change for this release, I'd prefer if the environment change were separated from the code fix.